### ISSUE TITLE
Cannot extract encrypted files when the password contains a german umlaut

### ISSUE THREAD

**Issue by glasen** [srikanth-lingala/zip4j#500]
Hi,

I've the problem that zip4j does not like umlauts (ä, ö, and ü) in a password. A normal password without any umlauts works fine. The zip-file can be extracted with all normal programs. Only zip4j cannot use the password.

**Comment by crusy** [comment]
Hi, I just tried with password "hällö-wörld" and it seems to work with v2.11.5. Can you verify?

**Comment by glasen** [comment]
Yes. Now it seems to work. Maybe the zip-file i've tested had some issues.
